examples/networking/router_route/main-route-to-compute.tf

```diff
@@ -0,0 +1,10 @@
+resource "vkcs_networking_router_route" "route_to_compute_example" {
```

Comment:
route_to_compute_example -> compute_gateway

Reply:
and rename the file to main.tf

Reply:
done